Backend/router/v1/report.js

```diff
@@ -2,17 +2,20 @@ const express = require('express');
 const router = express.Router();
 const reportCtrl = require('./report.ctrl');
 
-router.use((req, res, next) => {
-  next();
-});
+module.exports = () => {
+  router.use((req, res, next) => {
+    next();
+  });
 
-router.all('/', (req, res) => {
-  res.send("this is report root");
-});
+  router.all('/', (req, res) => {
+    res.send("this is report root");
```

Comment:
Fix English. And everywhere in the repository.

Reply:
@ethkim  I cant understand Fix English. Could you explain more?

Reply:
Update the message so people we better understand what it means. When I read 'this is report root', I have no idea what the situation is or what it means.